nautobot/core/views/mixins.py

```diff
@@ -436,9 +443,28 @@ class ObjectListViewMixin(NautobotViewSetMixin, mixins.ListModelMixin):
         "sort",  # table sorting
     )
 
+    def get_filtered_queryset_for_export(self):
```

Comment:
This seems like you're reinventing `filter_queryset()` which is something inherited from `GenericViewSet` that is a base for NautobotViewSetMixin`. 

See: https://github.com/encode/django-rest-framework/blob/master/rest_framework/generics.py#L140-L151

Just hoping to simplify here. Please correct me if I'm wrong!

Reply:
Thanks for pointing this out! Using it directly seems to cause some problems because the `export` query string passed in from the request isn't recognized as a filter for `NautobotFilterBackEnd` So I always get this error.
But it seems to be a good method to override and use.
```
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/django/test/utils.py", line 387, in inner
    return func(*args, **kwargs)
  File "/source/nautobot/utilities/testing/views.py", line 803, in test_queryset_to_csv
    self.assertHttpStatus(response, 200)
  File "/source/nautobot/utilities/testing/mixins.py", line 167, in assertHttpStatus
    self.assertIn(response.status_code, expected_status, err_message)
AssertionError: 400 not found in [200] : Expected HTTP status(es) [200]; received 400: {'export': [ErrorDetail(string='Unknown filter field', code=None)]}
```